Update r-server-storage.md

HeidiSteen opened the pull request.

azurecla commented:
Hi __@aashishr__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (aashishr). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, or work for Microsoft Open Technologies, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

deneha commented:
@aashishr Thanks for the contribution!  This link update is helpful, but unfortunately the link you inserted will not pass validation for localization purposes because of the /en-us/ portion of the link. Could you possibly make this change in the private repo?

deneha commented:
I've just checked the published version and the link update has been done in another pull request.  I'm going to close this request.

HeidiSteen committed:
Update r-server-storage.md

HeidiSteen committed:
Update r-server-storage.md

HeidiSteen committed:
Update r-server-storage.md

Per an open bug currently assigned to me, the call to RxSpark has to include nameNode param.
